LogMonitor/src/LogMonitor/LogWriter.h

```diff
@@ -51,7 +51,7 @@ public :
 
         AcquireSRWLockExclusive(&m_stdoutLock);
 
-        result = WriteFile(FileHandle, Buffer, NumberOfBytesToWrite, NumberOfBytesWritten, Overlapped);
+        result = WriteConsoleA(FileHandle, Buffer, NumberOfBytesToWrite, NumberOfBytesWritten, Overlapped);

```

Comment:
Can't find this `WriteConsoleA`  function, where is it defined? I'd think that for utf8 support, there would be a ...W (wide character) as opposed to ...A (ASCII)?

Reply:
Oh, found it in `consoleapi.h`; I think you should go with `WriteConsoleW`?

Reply:
And just to clarify @profnandaa ASCII ⊆  of Unicode. So WriteConsoleW should handle all cases.

Reply:
@profnandaa I have done some digging and this is the closest I've gotten to an explanation: https://github.com/microsoft/terminal/issues/396#issuecomment-477223265

WriteConsoleA on a console set to UTF8 will write out the entire utf-8 charset an not just ASCII characters. I've tested this with several non-Ascii characters and it works. Attempts to use `wchar_t, WCHAR and WriteConsoleW` have been unsuccessful.